docs/kb/semgrep-supply-chain/why-no-findings.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Troubleshoot why findings for Semgrep Supply Chain are not showing.
+tags:
+  - Semgrep Supply Chain
+---
+# Why aren't Supply Chain findings showing?
+
+## Ensure compatibility
+
+First, verify that your repository meets the basic requirements for Semgrep Supply Chain:
+
+### Are the language and ecosystem supported?
+
+Check the [Supported Languages table](/docs/supported-languages/#general-availability) for support for the project's language and ecosystem, as well as any ecosystem-specific requirements.
```

Comment:
```suggestion
Check the [Supported Languages table](/docs/supported-languages/#general-availability) to verify the project's language and ecosystem are supported, as well as any ecosystem-specific requirements.
```

Reply:
How about "to verify support for" - keep it short but remove the weird phrasing (good catch, ty).